src/js/utils/time.js

```diff
@@ -23,5 +31,5 @@ export function formatTime(seconds, guide) {
   // Check if leading zero is need for seconds
   s = s < 10 ? '0' + s : s;
 
-  return h + m + s;
+  return (negative ? '-' : '') + h + m + s;
```

Comment:
this diff is begging for test coverage, which is on the short list of things to set up 😄